br/pkg/lightning/errormanager/errormanager.go

```diff
@@ -94,7 +95,8 @@ const (
 			raw_handle  mediumblob NOT NULL COMMENT 'the data handle derived from the conflicted key or value',
 			raw_row     mediumblob NOT NULL COMMENT 'the data retrieved from the handle',
 			KEY (task_id, table_name),
-			INDEX (index_name)
+			INDEX (index_name),
```

Comment:
just noticed in line 97 we use KEY as the keyword for index, for one style we should unify them